Resfeber/Helpers/UIColor+Extension.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UIColor+Extension.swift
+//  Resfeber
+//
+//  Created by Joshua Rutkowski on 11/6/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import UIKit
+
+extension UIColor {
```

Comment:
This is a great idea! Something you might even like better is if you subclass UIColor and put all of your custom colors in there. That way instead of accessing a color by calling `UIColor.Resfeber.red`, you could just call something like `RFColor.red`. Is use the `RF` prefix because generally it is a good practice to give all of your file names for a given project a prefix. That way if the app grows enough or you have a Mac project, you can differentiate them by their prefix. At Day One we use (DO, DOI, DOX) to help us distinguish between iOS files, core files, and Mac files.

Reply:
That's a really good idea